tests/tests_lite/helpers/utils.py

```diff
@@ -0,0 +1,28 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
```

Comment:
for all the files kn tests: same concerns on git history

Reply:
what do you mean by concerns on git history?

Reply:
Only parts of the test files could be moved. Some of the tests still need to stay in PL until more classes move over. So git history can't be preserved, sorry :(